demisto_sdk/commands/format/tests/test_formatting_yml_test.py

```diff
@@ -1531,6 +1531,66 @@ def test_update_deprecate_in_playbook(
         assert base_update_yml.data["tests"] == [NO_TESTS_DEPRECATED]
         assert base_update_yml.data["description"] == description_result
 
+    @pytest.mark.parametrize(
+        "user_input, description_result",
+        [
+            ("", "Deprecated. No available replacement."),
+            ("Replacement entity", "Deprecated. Use Replacement entity instead."),
+        ],
+    )
+    def test_update_deprecate_in_modeling_rules(
+        self, pack, mocker, monkeypatch, user_input, description_result
+    ):
+        """
+        Given
+            - An playbook yml to deprecate.
```

Comment:
```suggestion
            - A playbook yml to deprecate.
```
Let's address the two cases specifically (case a: no user input, Then: expted output should be blabla)